Nodejs/MigrateToJsps/Ntvs/NjsprojFileReader.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media.Imaging;
+using System.Xml.Serialization;
+
+namespace MigrateToJsps
+{
+    internal class NjsprojFileReader
+    {
+        //TODO: change return type to void
+        public static NjsprojFileModel ProcessNjsproj(string njsprojPath)
+        {
+            var name = "";
+            var startupFile = "";
+            var projectTypeGuidsString = "";
+            var nodejsPort = "";
+
+            using (var fileStream = File.Open(njsprojPath, FileMode.Open, FileAccess.Read))
+            {
+                XmlSerializer xmlSerializer = new XmlSerializer(typeof(Project));
```

Comment:
`var` for consistency (`name` above is initialized with `var` for example).

Reply:
done